v5.5.sed

```diff
@@ -0,0 +1,2 @@
+s/@@VERSION@@/5.5/
```

Comment:
The issue with this simple script is what if you need literal @@VERSION@@ somewhere in the generated output. What if the result of a rule is a string that will be be matched by another rule by an accident?

While this seems pretty improbable, it creates a minefield that can hurt you in the future. I still think a real template system is necessary.